15_0_Setting_Up_a_Bitcoin_Regtest.md

```diff
@@ -0,0 +1,18 @@
+# Chapter 15: Setting Up a Bitcoin Regtest
+
+While developing Bitcoin applications, you might want to use your applications isolated from any public Blockchain such as the Mainnet or the Testnet.
```

Comment:
Remove `the` from `Mainnet` / `Testnet` and don't capitalize. (`such as mainnet or testnet`). This applies to the remainder of this PR (i.e. below 'the Regtest', etc).